rules-reviewed/core.windup.categories.xml

```diff
@@ -15,24 +15,34 @@
         <description>Items in this category will need to be assessed manually to determine whether or not action is required.</description>
     </category>
 
-    <category id="cloud-mandatory" priority="1010">
+    <category id="cloud-mandatory" priority="10000">
         <name>Mandatory for a cloud environment</name>
         <description>Items that will require changes in order for the migration to be successful.</description>
     </category>
 
-    <category id="cloud-optional" priority="2010">
+    <category id="cloud-optional" priority="11000">
         <name>Optional for a cloud environment</name>
         <description>Items that are recommended, but are not necessarily required for a successful migration.</description>
     </category>
 
-    <category id="openshift-mandatory" priority="1020">
+    <category id="cloud-potential" priority="12000">
```

Comment:
I think it's better to have also the optional categories for cloud and openshift.
I could have added it by myself but i was too focused on the categories discussed and i didn't think also about the potential one.